records/labels/labels_test.go

```diff
@@ -47,7 +47,8 @@ func TestRFC952(t *testing.T) {
 		"89fdgsf---gs7-fgs--d7fddg---123":           "fdgsf---gs7-fgs--d7fddg1",
 		"89fdgsf---gs7-fgs--d7fddg-":                "fdgsf---gs7-fgs--d7fddg",
 		"chronos with a space AND MIXED CASE-2.0.1": "chronoswithaspaceandmixe",
-		"chronos with a space AND----------MIXED--": "chronoswithaspaceandmixe",
+		"chronos with a space AND----------MIXED--": "chronoswithaspaceand---m",
```

Comment:
@sargun want to make sure that you see this line because it indicates a change in terms of how names are generated. i think it's an edge case.